src/pointer.cr

```diff
@@ -405,7 +405,7 @@ struct Pointer(T)
   # ptr.address # => 5678
   # ```
   def self.new(address : Int)
-    new address.to_u64
+    new LibC::SizeT.new(0)
```

Comment:
May be this way: https://github.com/crystal-lang/crystal/pull/4594#issuecomment-309394082 ?